From 1a37c69d92195b3f864ae89b4c9488ce8c961b31 Mon Sep 17 00:00:00 2001
From: trickypr <trickypr@icloud.com>
Date: Tue, 21 Jun 2022 15:26:35 +1000
Subject: [PATCH] Remove settings sync service

This service downloads settings from mozilla's servers and applies them to the local
gecko instance, which we do not want. This was causing errors with settings dumps earlier.
This commit might allow the removal of `0001-Build-Error-Remote-settings-dump-were-missing`
---
 services/moz.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/services/moz.build b/services/moz.build
index 4ef95d88d209..e460b94ee377 100644
--- a/services/moz.build
+++ b/services/moz.build
@@ -11,7 +11,6 @@ DIRS += [
     "common",
     "crypto",
     "interfaces",
-    "settings",
 ]
 
 # The automation dir is only included in nightlies and debug
-- 
2.36.1

